src/cli/src/commands/upload-build.ts

```diff
@@ -76,7 +76,7 @@ export const handler = async (args: Args): Promise<void> => {
       });
 
       res.on('end', () => {
-        const response = Buffer.from(output).toJSON();
```

Comment:
The response encoding is set to `utf8` (above), not `binary`, so I don't think it makes sense to use buffers here?